test/op_shape_test.cpp

```diff
@@ -2166,7 +2166,7 @@ TEST_CASE(multinomial)
     migraphx::shape s1{migraphx::shape::float_type, {1, 2}};
     migraphx::shape s2{migraphx::shape::float_type, {3, 4}};
     migraphx::shape s3{migraphx::shape::float_type, {1, 4}};
-    int dtype = 2;
+    int dtype = 3;
```

Comment:
These should be using the `migraphx::shape`. Should work with `migraphx::shape::float_type`.

Reply:
We should not be changing the value of `float_type`.